Plan/common/src/main/resources/assets/plan/web/css/sb-admin-2.css

```diff
@@ -1,9640 +1,1926 @@
+.table-success,
+.table-success > th,
+.table-success > td {
+    background-color: #bff0de;
 }
 
-small {
-    font-size: 80%;
+.table-success th,
+.table-success td,
+.table-success thead th,
+.table-success tbody + tbody {
+    border-color: #89e2c2;
 }
 
-sub,
-sup {
-    position: relative;
-    font-size: 75%;
-    line-height: 0;
-    vertical-align: baseline;
+.table-hover .table-success:hover {
+    background-color: #aaebd3;
 }
 
-sub {
-    bottom: -.25em;
+.table-hover .table-success:hover > td,
+.table-hover .table-success:hover > th {
+    background-color: #aaebd3;
 }
 
-sup {
-    top: -.5em;
+.table-info,
+.table-info > th,
+.table-info > td {
+    background-color: #c7ebf1;
 }
 
-a {
-    color: #4e73df;
-    text-decoration: none;
-    background-color: transparent;
+.table-info th,
+.table-info td,
+.table-info thead th,
+.table-info tbody + tbody {
+    border-color: #96dbe4;
 }
 
-a:hover {
-    color: #224abe;
-    text-decoration: underline;
+.table-hover .table-info:hover {
+    background-color: #b3e4ec;
 }
 
-a:not([href]):not([class]) {
-    color: inherit;
-    text-decoration: none;
+.table-hover .table-info:hover > td,
+.table-hover .table-info:hover > th {
+    background-color: #b3e4ec;
 }
 
-a:not([href]):not([class]):hover {
-    color: inherit;
-    text-decoration: none;
+.table-warning,
+.table-warning > th,
+.table-warning > td {
+    background-color: #fceec9;
 }
 
-pre,
-code,
-kbd,
-samp {
-    font-family: SFMono-Regular, Menlo, Monaco, Consolas, "Liberation Mono", "Courier New", monospace;
-    font-size: 1em;
+.table-warning th,
+.table-warning td,
+.table-warning thead th,
+.table-warning tbody + tbody {
+    border-color: #fadf9b;
 }
 
-pre {
-    margin-top: 0;
-    margin-bottom: 1rem;
-    overflow: auto;
-    -ms-overflow-style: scrollbar;
+.table-hover .table-warning:hover {
+    background-color: #fbe6b1;
 }
 
-figure {
-    margin: 0 0 1rem;
+.table-hover .table-warning:hover > td,
+.table-hover .table-warning:hover > th {
+    background-color: #fbe6b1;
 }
 
-img {
-    height: auto;
-    vertical-align: middle;
-    border-style: none;
+.table-danger,
+.table-danger > th,
+.table-danger > td {
+    background-color: #f8ccc8;
 }
 
-svg {
-    overflow: hidden;
-    vertical-align: middle;
+.table-danger th,
+.table-danger td,
+.table-danger thead th,
+.table-danger tbody + tbody {
+    border-color: #f3a199;
 }
 
-table {
-    border-collapse: collapse;
+.table-hover .table-danger:hover {
+    background-color: #f5b7b1;
 }
 
-caption {
-    padding-top: 0.75rem;
-    padding-bottom: 0.75rem;
-    color: #858796;
-    text-align: left;
-    caption-side: bottom;
+.table-hover .table-danger:hover > td,
+.table-hover .table-danger:hover > th {
+    background-color: #f5b7b1;
 }
 
-th {
-    text-align: inherit;
-    text-align: -webkit-match-parent;
+.table-light th,
+.table-light td,
+.table-light thead th,
+.table-light tbody + tbody {
+    border-color: #fbfcfd;
 }
 
-label {
```

Comment:
This change requires a Datatables.js update, or alternatively edits to Datatables.js itself which would also require us to use the non-minified version of the library. The `Responsive` module isn't yet updated for BS5 which keeps me from doing the update.

Reply:
This comment is about the whitespace below the top controls
![image](https://user-images.githubusercontent.com/24460436/114260671-0ecced80-99df-11eb-8696-acca4ed111b4.png)

Reply:
Yes. In Bootstrap 4.x, `label` elements used to have a .5rem bottom margin, which was moved to a separate `.form-label` class in BS5. The `Show x entries` arrow is also misplaced, because 5.0 dropped the `.custom-select` class. I can add our own CSS to the `label` element and the `.custom-select` class, and although it fixes the issue, IMO it's not a permanent fix and should be removed once we're able to update Datatables to properly support BS5.

Reply:
Looks like DataTables has css for bootstrap 5 beta1 available https://datatables.net/examples/styling/bootstrap5